replica-common/src/main/java/com/graphhopper/OsmHelper.java

```diff
@@ -1,18 +1,41 @@
 package com.graphhopper;
 
+import com.google.common.collect.Lists;
+import com.google.common.collect.Maps;
+import com.google.common.collect.Sets;
 import com.graphhopper.reader.ReaderElement;
+import com.graphhopper.reader.ReaderRelation;
 import com.graphhopper.reader.ReaderWay;
+import com.graphhopper.reader.osm.OSMInput;
+import com.graphhopper.reader.osm.OSMInputFile;
 import com.graphhopper.storage.DataAccess;
 import com.graphhopper.util.BitUtil;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 
-import java.util.Map;
+import java.io.File;
+import java.util.*;
 
 public class OsmHelper {
+    private static final Logger LOG = LoggerFactory.getLogger(OsmHelper.class);
+
     private DataAccess nodeMapping;
     private DataAccess artificialIdToOsmNodeIdMapping;
     private DataAccess ghEdgeIdToSegmentIndexMapping;
     private BitUtil bitUtil;
 
+    public static final String OSM_NAME_TAG = "name";
+    public static final String OSM_HIGHWAY_TAG = "highway";
+    public static final String OSM_LANES_TAG = "lanes";
+    public static final String OSM_FORWARD_LANES_TAG = "lanes:forward";
+    public static final String OSM_BACKWARD_LANES_TAG = "lanes:backward";
+
+    // Tags we consider when calculating the value of the `lanes` column
+    public static final Set<String> LANE_TAGS = Sets.newHashSet(OSM_LANES_TAG, OSM_FORWARD_LANES_TAG, OSM_BACKWARD_LANES_TAG);
+    // Tags we parse to include as columns in network link export
+    public static final Set<String> WAY_TAGS = Sets.newHashSet(OSM_HIGHWAY_TAG);
+    public static final Set<String> ALL_TAGS_TO_PARSE = Sets.union(LANE_TAGS, WAY_TAGS);
```

Comment:
nit: looks like we also parse `OSM_NAME_TAG`, so this set doesn't contain all parsed tags?

Reply:
Ah, good callout - I decided to do a bit more refactoring in [4f061ff](https://github.com/replicahq/graphhopper/pull/161/commits/4f061ff477e694fdfc9b4f298322c166ed87f205) to address this + make things a bit more flexible for the fast-follow change that parses the new columns. Now these collections make more sense + we're able to use one function to parse any set of way or relation tags